app/javascript/menu/side-nav-menu-link.jsx

```diff
@@ -24,10 +24,7 @@ export const SideNavMenuLink = ({
 
   return (
     <SideNavItem id={id}>
-      <Link
-        className={className}
-        onClick={onClick}
-      >
+      <Link className={className} onClick={onClick} onKeyPress={onClick} tabindex="0">
```

Comment:
@skateman `tabindex` here should be `tabIndex` i am seeing a warning in console `Warning: Invalid DOM property `tabindex`. Did you mean `tabIndex`?`
Other than that it works.